pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -54,8 +54,8 @@ func NewStorageFactoryConfig() *StorageFactoryConfig {
 
 	resources := []schema.GroupVersionResource{
 		batch.Resource("cronjobs").WithVersion("v1beta1"),
-		networking.Resource("ingresses").WithVersion("v1beta1"),
-		networking.Resource("ingressclasses").WithVersion("v1beta1"),
+		networking.Resource("ingresses").WithVersion("v1"),
+		networking.Resource("ingressclasses").WithVersion("v1"),
```

Comment:
revert this change, we cannot switch the storage version to v1 until 1.20 (1.19 has to remain compatible with 1.18 API servers that don't know how to read v1 out of etcd)